store/store.go

```diff
@@ -1411,7 +1413,7 @@ func (s *Store) Download(ctx context.Context, name string, targetPath string, do
 	return s.cacher.Put(downloadInfo.Sha3_384, targetPath)
 }
 
-func reqOptions(storeURL *url.URL, cdnHeader string) *requestOptions {
+func reqOptions(storeURL *url.URL, cdnHeader string, opts *DownloadOptions) *requestOptions {
```

Comment:
again not related to this, but the change make clear that this function should mention GET somehow in its name

Reply:
hah, when I saw it I thought "hmm, this should probably say Download somewhere in its name"